tests/acceptance/tests/basic_dsc_resources/waitforall/negative/waitforall_blacklist.rb

```diff
@@ -18,7 +18,7 @@
 dsc_manifest = ERB.new(File.read(dsc_manifest_template_path), 0, '>').result(binding)
 
 # Verify
-error_msg = /Error:.*Invalid resource type dsc_waitforall/
+error_msg = /Error:.*resource.*dsc_waitforall/
```

Comment:
Makes sense. I was just wondering if there would be some other error that bubbles up that is masked by the regex, but I think that's an outside chance given what we're talking about here. 👍